conf/log4j2-dev.xml

```diff
@@ -1,136 +1,186 @@
-    </Logger>
-    <Logger name="org.onehippo.cms7.services.autoreload.AutoReloadServiceModule" additivity="false" level="info">
-      <AppenderRef ref="messages"/>
-    </Logger>
-    <Logger name="org.hippoecm.hst.core.container.WebFileValve" additivity="false" level="info">
-      <AppenderRef ref="messages"/>
-    </Logger>
-
-    <!-- project logging -->
-    <Logger name="uk.nhs.digital.ps" level="debug"/>
-
-    <Root level="warn">
-      <AppenderRef ref="site"/>
-      <AppenderRef ref="cms"/>
-      <AppenderRef ref="console"/>
-    </Root>
-
-  </Loggers>
+    <Appenders>
+
+        <!-- hippo-cms.log -->
+        <RollingFile name="cms" fileName="${sys:catalina.base}/logs/hippo-cms.log"
+                                 filePattern="${sys:catalina.base}/logs/hippo-cms.log.%d{yyyy-MM-dd}">
+            <LookupFilter key="jndi:logging/contextName" value="cms" onMatch="ACCEPT"/>
+            <PatternLayout pattern="%d{dd.MM.yyyy HH:mm:ss} %-5p %t [%C{1}.%M:%L] %m%n"/>
+            <Policies>
+                <TimeBasedTriggeringPolicy/>
+            </Policies>
+        </RollingFile>
+
+        <!-- hippo-site.log -->
+        <RollingFile name="site" fileName="${sys:catalina.base}/logs/hippo-site.log"
+                                 filePattern="${sys:catalina.base}/logs/hippo-site.log.%d{yyyy-MM-dd}">
+            <LookupFilter key="jndi:logging/contextName" value="site" onMatch="ACCEPT"/>
+            <PatternLayout pattern="%d{dd.MM.yyyy HH:mm:ss} %-5p %t [%C{1}.%M:%L] %m%n"/>
+            <Policies>
+                <TimeBasedTriggeringPolicy/>
+            </Policies>
+        </RollingFile>
+
+        <!-- hippo-audit.log -->
+        <RollingFile name="audit" fileName="${sys:catalina.base}/logs/hippo-audit.log"
+                                 filePattern="${sys:catalina.base}/logs/hippo-audit.log.%d{yyyy-MM-dd}">
+            <PatternLayout pattern="%d{dd.MM.yyyy HH:mm:ss} %-5p %t [%C{1}.%M:%L] %m%n"/>
+            <Policies>
+                <TimeBasedTriggeringPolicy/>
+            </Policies>
+        </RollingFile>
+
+        <!-- console -->
+        <Console name="console" target="SYSTEM_OUT">
+            <PatternLayout pattern="%d{dd.MM.yyyy HH:mm:ss} %-5p %t [%C{1}.%M:%L] %m%n"/>
+        </Console>
+
+        <!-- messages -->
+        <Console name="messages" target="SYSTEM_OUT">
+            <PatternLayout pattern="%d{HH:mm:ss} %-5p %m%n"/>
+        </Console>
+
+        <!-- warnings -->
+        <Console name="warnings" target="SYSTEM_ERR">
+            <PatternLayout pattern="%%d{HH:mm:ss} %-5p [%C{1}] %m%n"/>
+        </Console>
+
+        <!-- splunk CMS -->
+        <Http name="splunk-cms"
+            url="${sys:splunk.url}"
+            token="${sys:splunk.token}"
+            batch_size_count="${sys:splunk.batch.size.count}"
+            disableCertificateValidation="true"
+            source="nhsdhippocms"
+            sourcetype="hippo-cms">
+            <PatternLayout pattern="%m"/>
```

Comment:
@ngta1 can you test the "<LookupFilter ...>" tag. This might simplify the config. Check the hippo-cms and hippo-site files appenders above.